### PR TITLE
Small edits for H4CC landing page

### PR DESCRIPTION
**Pull request by patphongs**
## Summary (required)

Small edits that are needed to the Help for candidates and committees landing page based on comments from @jenniferthibault, @johnnyporkchops, and @dorothyyeager. This includes:

- Fixing the text for the `Statements of organization filed by committees` link under you might also like
- Updating the styling for the All tips button
- Adding the additional disabled link for `Fundraising for an SSF` to the Corporations and labor organizations section of the guides page. This is so we can match what is currently in production.

## Impacted areas of the application
- Help for candidates and committees landing page
- Guides page

### You might also like section inside Help for candidates and committees landing
screen shot 2017-11-16 at 3 34 57 pm

### All tips button style update
screen shot 2017-11-16 at 3 36 00 pm

### Guides page
screen shot 2017-11-16 at 3 37 08 pm

### CODE REVIEW

**Comment by xtine** [comment]
Can you add the breadcrumbs on Guides page as found on this issue: https://github.com/18F/fec-cms/issues/1502 ?

**Comment by xtine** [comment]
Line 77: `<h3 class="u-no-margin"><a href="/help-candidates-and-committees/candidate-taking-receipts/contribution-limits-candidates/">Contribution limits</a></h3>`
Remove `class="u-no-margin"` for proper design spacing of that heading.

**Comment by xtine** [comment]
For the Candidates and Committees landing page, the `.example--primary` section should be 1.6 font size. I talked to @jenniferthibault and all elements of that type (section with left border) should be 1.6, so go ahead and commit in a change to add `font-size: u(1.6rem)` for `.example--primary`.

**Comment by xtine** [comment]
On the "Tips for Treasurers" feed, the date needs to have a `$gray-dark` color. Since this convention will be reused for other pages, I would suggest making a new class called `.date--color` with just the color attribute on `components/_posts.scss`.

**Comment by jenniferthibault** [comment]
[Line 78:](https://github.com/18F/fec-cms/pull/1504/files#diff-b5fca762a031c4182e25a1859f97e9c2R78) Should be `Federal Election Campaign Act` instead of `Federal Election Campaign Finance Act` (Remove the word `Finance`). Lauren or Rebecca pointed this out at design pairing

File: fec/fec/static/scss/components/_posts.scss [hunk 1]
```diff
@@ -177,3 +177,11 @@
     font-size: u(1.4rem);
   }
 }
+
+// News Feeds
+// Class used to change date color on news feeds that contain the date next to the link title in the feed.
+// For example: The Tips for treasurers news feed on the Help for candidates and committees landing 
+
+.date--gray-dark {
```
Comment (xtine) [inline fec/fec/static/scss/components/_posts.scss]: I hesitate to explicitly name this class as gray-dark. Perhaps even more generically, calling it `.feed-date`?

Comment (patphongs) [reply fec/fec/static/scss/components/_posts.scss]: Thanks @xtine, I agree. I also wanted to practice more of the BEM conventions, so I changed it to `.feed--list__date`

**Comment by xtine** [comment]
Per design pairing, I have updated the Guides page with fixes required.

**Comment by xtine** [comment]
@patphongs: the last thing we need is just for you to review that everything looks good and this should be ready to go!

File: fec/home/templates/home/collection_page.html [hunk 2]
```diff
@@ -81,7 +81,7 @@ <h2 class="sidebar__title sidebar__title--large icon-heading--financial-document
             <span class="card__icon i-question-bubble"><span class="u-visually-hidden">Icon representing a question</span></span>
         </div>
         <div class="card__content">
-          Get help from the FEC by  phone or email
+          Get help from the FEC by phone or email
```
Comment (jenniferthibault) [inline fec/home/templates/home/collection_page.html]: @xtine @patphongs what happened here? It looks like it took out a space?

File: fec/home/templates/home/resource_page.html [hunk 3]
```diff
@@ -73,7 +73,7 @@ <h2>Related topics</h2>
               <span class="card__icon i-question-bubble"><span class="u-visually-hidden">Icon representing a question</span></span>
           </div>
           <div class="card__content">
-            Get help from the FEC by  phone or email
+            Get help from the FEC by phone or email
```
Comment (jenniferthibault) [inline fec/home/templates/home/resource_page.html]: Same question about disappearing space

File: fec/home/templates/home/collection_page.html (hunk 2 above)
Comment (xtine) [reply fec/home/templates/home/collection_page.html]: For some reason the merge included those files? Super weird, sorry that happened. Let's revert those changes.

**Pull request merged**